app/assets/stylesheets/modules/forms.scss

```diff
@@ -3,8 +3,19 @@
     padding-top: 7px;
   }
 
+  hr {
+    margin-top: 20px;
+    margin-bottom: 20px;
+    border: 0;
+    border-top: 1px solid #d5d5d3;
+  }
+
   .or-divider {
     text-align: center;
+    width: 3em;
+    margin: -32px auto 10px auto;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `-32px auto 10px` instead of `-32px auto 10px auto`